third_party/move/move-compiler-v2/tests/unused-assignment/v1-locals/unused_resource_explicit_return.exp

```diff
@@ -0,0 +1,39 @@
+
+Diagnostics:
+warning: Unused local variable `r`. Consider removing or prefixing with an underscore: `_r`
+  ┌─ tests/unused-assignment/v1-locals/unused_resource_explicit_return.move:5:13
+  │
+5 │         let r = R{};
+  │             ^
+
+warning: Unused local variable `r`. Consider removing or prefixing with an underscore: `_r`
+   ┌─ tests/unused-assignment/v1-locals/unused_resource_explicit_return.move:28:13
+   │
+28 │         let r = R{};
+   │             ^
+
+warning: Unused local variable `x`. Consider removing or prefixing with an underscore: `_x`
+   ┌─ tests/unused-assignment/v1-locals/unused_resource_explicit_return.move:33:13
+   │
+33 │         let x = &R{};
+   │             ^
+
+
+Diagnostics:
+warning: Unused assignment to `r`. Consider removing or prefixing with an underscore: `_r`
```

Comment:
Hmm. Looks like we are duplicating warnings (although they are of different "types", to a user, they are very similar). What does v1 do in these cases? IMO, we should not have a unused assignment warning if for the same location we have already provided unused variable warning.

Reply:
This should be avoided, but the solution is probably not trivial.